src/test/java/org/opensearch/flowframework/FlowFrameworkRestTestCase.java

```diff
@@ -119,8 +119,19 @@ public void setUpSettings() throws Exception {
             );
             assertEquals(200, response.getStatusLine().getStatusCode());
 
+            // Set ML native memory threshold to 100 to avoid opening the circuit breaker during tests
+            response = TestHelpers.makeRequest(
+                client(),
+                "PUT",
+                "_cluster/settings",
+                null,
+                "{\"persistent\":{\"plugins.ml_commons.native_memory_threshold\":100}}",
```

Comment:
Should this be `transient` setting for the cluster?

Reply:
It can be either, transient vs persistent really only matters during cluster restarts. In our case, we just spin up the cluster once and run all our tests before terminating the cluster